### ISSUE TITLE
Arena Grand Master

### ISSUE THREAD

**Issue by rustndust** [Atlantiss/BugTracker#8164]
Hey,

Stranglethorn Vale, Arena Grand Master quest. Done this several times on other toons in the past, but I have a problem with the current one. I could pick up only the first trinket. After that five next times the only thing I found in the chest was some junk. My alt, who's killing guards is able to open the chest after my twink does so, and pick up the trinkets. This is very frustrating. I tried to put the first one in my bank - no effect. The situation for now is I have 5 trinkets on a toon who doesn't even need them and only the first one on my twink. Wasted some time pointlessly and ran out of options. Please help me. I know u're busy, but if there is a simple fix, please do so. I don't know whether i should try again and risk gathering trinkets on the wrong toon or wait for a fix. 

Cheers.

**Comment by Sand3r-** [comment]
Try to kill the toon that you are trying to aquire the trinket with, and try again (that's what has helped me when I had the same problem)

**Comment by rustndust** [comment]
Tried to do so this time. As always: killed guards with my alt, then my twink opened the chest. Junk as always, but this time enchanting recipe, a bag and sth else. No usual healing potions or arena wrists. Killed my twink and then I realised the chest had disappeared. There was no trinket in it at all. Otherwise it would be standing there containing only the trinket (like every other time when my alt was able to re-open it after my twink). Major messup there. Think I'll try your solution again next time, maybe it was an one-time bug. At least I hope so...

**Comment by Aival94** [comment]
Fix your tittle and give some links as well.

**Comment by rustndust** [comment]
What's wrong with the title? I can fix it, sure, i just need to know what to fix.

Screenies:

1. Twink opens the chest and this is what he finds:
[http://i.imgur.com/iR0Z9sh.jpg](url)

2. Then the other toon opens it and picks up the trinket w/o problems.
[http://imgur.com/wIYBwVW.jpg](url)

**Comment by Aival94** [comment]
Title should be like they said in [Readme](https://github.com/Atlantiss/BugTracker/blob/master/README.md)

**Comment by rustndust** [comment]
I'm pretty sure Arena Grand Master means only one thing but ok, fixed that.

**Comment by rustndust** [comment]
Umm... bump..?

**Comment by Noviush** [comment]
It's still bugged, and it's a little frustrating, when old twinks on 14th lvl have this op trinket, and new one have no possibility to get it...

**Comment by rustndust** [comment]
True. 2 CTA's (AB and WSG) lately and not having the AGM was a pain there...

**Comment by Samineh** [comment]
Bump
